packages/expo-updates/ios/EXUpdates/UpdatesUtils.swift

```diff
@@ -133,14 +133,22 @@ public final class UpdatesUtils: NSObject {
     }
   }
 
-  internal static func isNativeDebuggingEnabled() -> Bool {
+  public static func isNativeDebuggingEnabled() -> Bool {
 #if EX_UPDATES_NATIVE_DEBUG
     return true
 #else
     return false
 #endif
   }
 
+  internal static func isUsingCustomInitialization() -> Bool {
```

Comment:
```suggestion
  internal static func useCustomInitialization() -> Bool {
```

maybe just have "use" prefixed convention

Reply:
Sure! There are other variables like `isUsingEmbeddedAssets` that we might want to adjust as well, I'll take a look at the naming

Reply:
boolean method names generally should begin with a question, so `isBlah`, `hasBlah`, `shouldUseBlah`, `isUsingBlah`. I think what you have originally is good.

Reply:
Ok I've hopefully made everything consistent and starting with "isUsing"...

Reply:
ok consistency is great!